qiskit_research/utils/pulse_scaling.py

```diff
@@ -325,13 +325,26 @@ def cr_scaling_passes(
 
     yield TemplateOptimization(**templates)
     yield CombineRuns(["rzx"])
-    # pauli twirl here
     yield RZXtoEchoedCR(inst_sched_map)
     yield Optimize1qGatesDecomposition(BASIS_GATES)
     yield CXCancellation()
     yield CombineRuns(["rz"])
     if param_bind is not None:
+        # instead try:
+        # yield pulse_attaching_passes(inst_sched_map, channel_map, param_bind)
```

Comment:
What's the meaning of these comments? I think this should work if you use `yield from`:
```python
yield from pulse_attaching_passes(inst_sched_map, channel_map, param_bind)
```